packages/react/src/primitives/Button/__tests__/Button.test.tsx

```diff
@@ -46,6 +54,94 @@ describe('Button test suite', () => {
     expect(menu.classList).toContain(`${ComponentClassNames['Button']}--menu`);
   });
 
+  it.each(SUPPORTED_COLOR_THEMES)(
+    'should render the %s color theme for the default variation',
+    async (colorTheme) => {
+      const testId = `default-${colorTheme}-ColorTheme`;
+      render(
+        <Button
+          testId={testId}
+          colorTheme={colorTheme as ButtonColorTheme}
```

Comment:
_Think_ we can avoid casting here by annotating `SUPPORTED_COLOR_THEMES` with `ButtonColorTheme[]`

```
const SUPPORTED_COLOR_THEMES: ButtonColorTheme = [];
```

Reply:
Yes, that works. Updated.